AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/ModelSyncedEvent.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+/// Used as HubPayload for the `ModelSynced`
+/// - `modelName` (String): the name of the model that was synced
+/// - `isFullSync` (Bool): `true` if the model was synced with a "full" query to retrieve all models
+/// - `isDeltaSync` (Bool): `true` if the model was synced with a "delta" query to retrieve changes since the last sync
+/// - `createCount` (Int): the number of new model instances added to the local store
+/// - `updateCount` (Int): the number of existing model instances updated in the local store
+/// - `deleteCount` (Int): the number of model instances deleted from the local store
+public struct ModelSyncedEvent {
+    public let modelName: String
+    public let isFullSync: Bool
+    public let isDeltaSync: Bool
+    public let createCount: Int
+    public let updateCount: Int
+    public let deleteCount: Int
```

Comment:
from my understanding, we agreed to follow JS unless Android has differed from JS, so we follow Android for native parity. If this is true for this case, can you double confirm if you should use 
```
createCount
updateCount
deleteCount
```
or follow Android's https://github.com/aws-amplify/amplify-android/blob/main/core/src/main/java/com/amplifyframework/datastore/events/ModelSyncedEvent.java#L29-L31
```
added
updated
deleted
```

Reply:
updated